client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -67,8 +81,19 @@ export class PageViewTracker extends React.Component {
 	}
 }
 
+const mapStateToProps = state => {
+	const selectedSiteId = getSelectedSiteId( state );
+	const selectedSiteSlug = getSiteSlug( state, selectedSiteId );
+	const currentSlug = getSiteFragment( get( window, 'location.pathname', '' ) );
+	const hasSelectedSiteLoaded = ! currentSlug || currentSlug === selectedSiteSlug;
```

Comment:
@lamosty @jsnajdr are there any site conflict corner cases we should be aware of, or would the url normally redirect in that case?

Reply:
Hmm, not sure. Any redirects that happen on site selection level are defined in `my-sites/controller/siteSelection`